HomeEdu/Scenes/Main/Account/EditStudentInfo.swift

```diff
@@ -0,0 +1,160 @@
+                             for: .valueChanged)
+        let tapGesture = UITapGestureRecognizer(target: self,
+                                                action: #selector(EditStudentInfo.viewTapped(gestureRecognizer:)))
+        view.addGestureRecognizer(tapGesture)
+        birthdayTextField.inputView = datePicker
+    }
+    
+    @objc func viewTapped(gestureRecognizer: UITapGestureRecognizer) {
+        view.endEditing(true)
+    }
+    
+    @objc func dateChanged(dataPicker: UIDatePicker) {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd"
+        birthdayTextField.text = dateFormatter.string(from: datePicker.date)
+        view.endEditing(true)
+    }
+    
+    @IBAction func checkBoxMale(_ sender: Any) {
+        checkBoxMale.setImage(checkBoxImg, for: .normal)
+        checkBoxFemale.setImage(unCheckBoxImg, for: .normal)
+        sexValueChange = Constant.Sex.male
+    }
+    
+    @IBAction func checkBoxFemale(_ sender: Any) {
+        checkBoxMale.setImage(unCheckBoxImg, for: .normal)
+        checkBoxFemale.setImage(checkBoxImg, for: .normal)
+        sexValueChange = Constant.Sex.female
+    }
+    
+    @IBAction func buttonBack(_ sender: Any) {
+        self.navigationController?.popViewController(animated: true)
+    }
+    
+    @IBAction func buttonSave(_ sender: Any) {
+        if checkIfStudentInfoIsValid() {
+            confirmChangeInfo()
+        } else {
+            self.createAlert(title: NSLocalizedString(Constant.Alert.alertTitle, comment: ""),
+                             message: NSLocalizedString(Constant.Alert.editInfoError, comment: ""))
+        }
+    }
+    
+    func requestEditInfo() {
+        let param: Parameters = [
+            "fullname": fullnameTextField.text ?? "",
+            "birthday": birthdayTextField.text ?? "",
+            "sex": sexValueChange,
+            "other_email": otherMailTextField.text ?? "",
+            "nation": nationTextField.text ?? "",
+            "address": addressTextField.text ?? ""
+        ]
+        EditInfoRequest.shared.editInfo(param) {
+            self.isConnect = $0
+            self.isSuccessEdit = $1
+            if self.isConnect {
+                if self.isSuccessEdit {
+                    self.createAlert(title: NSLocalizedString(Constant.Alert.alertTitle, comment: ""),
+                                     message: NSLocalizedString(Constant.Alert.editInfoSuccess, comment: ""))
+                    self.isChange = true
+                } else {
+                    self.createAlert(title: NSLocalizedString(Constant.Alert.alertTitle, comment: ""),
+                                     message: NSLocalizedString(Constant.Alert.editInfoFail, comment: ""))
+                }
+            } else {
+                self.internetNotConnect()
+            }
+        }
+    }
+    
+    func checkIfStudentInfoIsValid() -> Bool {
```

Comment:
```suggestion
    func checkIfStudentInfoIsValid(parram here) -> Bool {
```
check thì phải có input chứ em

Reply:
Cái này em kiểm tra xem người dùng đã thay đổi thông tin hay chưa ạ